scripts/dump_load_db.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash -e
+
+cd $(dirname $0)
+
+# some config variables
+OUTPUT_DIR="$(realpath ${2:-../db_dumps})"
```

Comment:
This doesn't seem to work in OS X where `realpath` isn't standard. 

Maybe something like:
```suggestion
function path {
  if command -v realpath >/dev/null
  then
    realpath $1
  else
    echo "$1"
  fi
}

OUTPUT_DIR="$(path ${2:-../db_dumps})"
```